### PR TITLE
Adds craftable bracers, adds watchers dropping bones.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
For when you just KNOW that you're gonna get hit in the arms.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
ohon

**Comment by drynwyn** [comment]
Wouldn't these being fire proof prevent the wearer from burning their hands on a lightbulb? That doesn't really make sense since they're bracers that don't actually cover the hands.

**Comment by TechnoAlchemisto** [comment]
burn_state = fireproof means they won't burn in lava/fire.

**Comment by RandomMarine** [comment]
You know armor directly stacks with anything that covers the same parts?

A miner can easily get over 100 melee defense to their arms with these.

**Comment by MMMiracles** [comment]
@RandomMarine armor caps at 90, its been like that for awhile now.

**Comment by KazeEspada** [comment]
So glad... my arms are immune to damage... totally important, right?

**Comment by KorPhaeron** [comment]
@TechnoAlchemisto 

Lava actually burns fireproof things

**Comment by TechnoAlchemisto** [comment]
o oh well

**Comment by FantasticFwoosh** [comment]
You want the lavaproof variable techno for lavaproof. (The lava forces the fire to happen bypassing fireproofing) for reference you could observe the code for the recent atmos HE pipes or Kor's existing lava boats pr

**Comment by TechnoAlchemisto** [comment]
It's not really necessary, I doubt anyone's bracers will be falling into the lava anyhow.

**Comment by KorPhaeron** [comment]
Eyeballs have bones is poor science and we strive for realism here

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -35,4 +35,21 @@
...
+	item_color = null	//So they don't wash.
+	transfer_prints = TRUE
+	strip_delay = 40
+	put_on_delay = 20
+	body_parts_covered = ARMS
+	cold_protection = ARMS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	burn_state = FIRE_PROOF
+	armor = list(melee = 15, bullet = 35, laser = 35, energy = 20, bomb = 35, bio = 35, rad = 35) //Not like anything ever hits the arms anyways.
```
Comment (KorPhaeron) [inline code/modules/clothing/gloves/miscellaneous.dm]: That's still a ton of bullet/laser resist if you stacked it with HoS armour or something but I guess most things using guns are players actually targeting you

Comment (TechnoAlchemisto) [inline code/modules/clothing/gloves/miscellaneous.dm]: The fact of the matter is this 

1. This item is pretty much exclusively going to be used by miners and lavaland natives, the hassle of getting a pair from a miner as any job that could capitalize on the "MAX ARMOR" benefit of the arms isn't even really worth the effort.

2. You cannot wear insulated gloves and bracers, or any regular gloves.

3. Nobody ever hits the arms, it's just a small chance.

**Pull request merged**